zerver/lib/upload.py

```diff
@@ -78,9 +81,6 @@ def random_name(bytes: int=60) -> Text:
 class BadImageError(JsonableError):
     code = ErrorCode.BAD_IMAGE
 
-class ExceededQuotaError(JsonableError):
```

Comment:
hm, why is this going away?
okay, looks like it is not used anymore. I would make a separate commit that removes it.